real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java

```diff
@@ -152,7 +152,9 @@ private void setupSubscriberList() {
             new Subscriber(filter.getMsgPath(), new MessageListener() {
                 @Override
                 public void actionPerformed(String topicName, Message m) {
-                    messageQueue.add(m);
+                    if (keepLogging) {
```

Comment:
yes but we should set the buffer size so that does not happen and if it does then we should throw an exception.